src/celestia/view.cpp

```diff
@@ -262,8 +262,8 @@ void View::drawBorder(int gWidth, int gHeight, const Color &color, float linewid
 
 void View::updateFBO(int gWidth, int gHeight)
```

Comment:
maybe better to change signature of this method to accept `unsigned`? i can't imaging negative dimensions

Reply:
Probably, but doing that ends up with knock-on changes to celestiacore.